### DISCUSSION TITLE
[done] It's time to add the vlang-blog entry link on the V official website

### DISCUSSION THREAD

**Discussion by tzSharing** [vlang/vlang-blog#21] [General]
vlang-blog has gradually taken shape, it is time to add the vlang-blog entry link on the V official website.

**Comment by hungrybluedev** [comment]
@medvednikov @spytheman what do you think?

**Comment by hungrybluedev** [comment]
Link to the blog has been added to the main website!
![screenshot](https://user-images.githubusercontent.com/46458389/160337642-8896338d-081a-434d-9c3e-77b98bf46e4b.png)
